generators/hub/templates/package.json

```diff
@@ -1,27 +1,29 @@
 {
   "scripts": {
-    "build": "npm run lint && npm run clean && webpack --progress --colors --config webpack.config.js --output-path ./dist -p",
+    "build": "npm run lint && npm run clean && npm run webpack",
+    "webpack" : "webpack --progress --colors --config webpack.config.js --output-path ./dist -p",
     "postbuild": "npm run package",
     "package": "tfx extension create  --rev-version",
     "gallery-publish": "tfx extension publish --rev-version",
     "clean": "rimraf ./dist && rimraf ./*.vsix",
     "lint": "tslint -c tslint.json \"src/*.ts"
```

Comment:
This will generate an issue running the generator on macOS 

> sh: -c: line 0: unexpected EOF while looking for matching `"'
> sh: -c: line 1: syntax error: unexpected end of file

Fix should be 
"lint": "tslint -c tslint.json 'src/*.ts'"